<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-NORF524" failures="1" tests="2" name="Error Validation Test" time="12.485" errors="0" timestamp="2024-02-28T09:51:02 IST">
  <testcase classname="AutomationProject.JavaSeleniumFramework.ErrorValidation" name="LockedOutUser" time="1.316">
    <failure type="java.lang.IllegalArgumentException" message="Incorrect Error Message">
      <![CDATA[java.lang.IllegalArgumentException: Incorrect Error Message
at dev.failsafe.internal.util.Assert.isTrue(Assert.java:29)
at AutomationProject.JavaSeleniumFramework.ErrorValidation.LockedOutUser(ErrorValidation.java:27)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- LockedOutUser -->
  <testcase classname="AutomationProject.JavaSeleniumFramework.ErrorValidation" name="invalidLogin" time="0.571"/>
</testsuite> <!-- Error Validation Test -->
